packages/nimble-components/src/rich-text-editor/tests/rich-text-editor.spec.ts

```diff
@@ -10,4 +41,599 @@ describe('RichTextEditor', () => {
     it('should export its tag', () => {
         expect(richTextEditorTag).toBe('nimble-rich-text-editor');
     });
+
+    it('should initialize Tiptap editor', () => {
+        expect(pageObject.editorSectionHasChildNodes()).toBeTrue();
+        expect(pageObject.getEditorSectionFirstElementChildClassName()).toBe(
+            'ProseMirror'
+        );
+    });
+
+    it('should set aria role as "textbox"', () => {
+        const editor = element.shadowRoot?.querySelector('.editor');
+
+        expect(editor!.getAttribute('role')).toBe('textbox');
+    });
+
+    it('should set aria-multiline to true', () => {
+        const editor = element.shadowRoot?.querySelector('.editor');
+
+        expect(editor!.getAttribute('aria-multiline')).toBe('true');
+    });
+
+    it('should have a slot and part named "footer-actions"', () => {
+        expect(pageObject.getSlotName()).toBe('footer-actions');
+        expect(pageObject.getSlotPartName()).toBe('footer-actions');
+    });
+
+    it('should have either one of the list buttons checked at the same time on click', async () => {
+        await waitForUpdatesAsync();
+        expect(pageObject.getButtonCheckedState('bullet-list')).toBeFalse();
+        expect(pageObject.getButtonCheckedState('numbered-list')).toBeFalse();
+
+        await pageObject.clickBulletListButton();
+        expect(pageObject.getButtonCheckedState('bullet-list')).toBeTrue();
+        expect(pageObject.getButtonCheckedState('numbered-list')).toBeFalse();
+
+        await pageObject.clickNumberedListButton();
+        expect(pageObject.getButtonCheckedState('bullet-list')).toBeFalse();
+        expect(pageObject.getButtonCheckedState('numbered-list')).toBeTrue();
+    });
+
+    const formattingButtons: {
+        name: string,
+        className: string,
+        iconName: string,
+        shortcutKey: string,
+        shiftKey: boolean
+    }[] = [
+        {
+            name: 'bold',
+            className: 'bold',
+            iconName: 'NIMBLE-ICON-BOLD-B',
+            shortcutKey: 'b',
+            shiftKey: false
+        },
+        {
+            name: 'italics',
+            className: 'italics',
+            iconName: 'NIMBLE-ICON-ITALIC-I',
+            shortcutKey: 'i',
+            shiftKey: false
+        },
+        {
+            name: 'bullet-list',
+            className: 'bullet-list',
+            iconName: 'NIMBLE-ICON-LIST',
+            shortcutKey: '8',
+            shiftKey: true
+        },
+        {
+            name: 'numbered-list',
+            className: 'numbered-list',
+            iconName: 'NIMBLE-ICON-NUMBER-LIST',
+            shortcutKey: '7',
+            shiftKey: true
+        }
+    ];
+
+    describe('should initialize nimble toggle buttons with necessary settings', () => {
```

Comment:
I'd consider this test to be overly detailed too. The button appearance, content hidden state, and icon are covered by the chromatic visual tests.

Reply:
Removed these test cases as well.